src/components/auth/Login.js

```diff
@@ -18,7 +12,7 @@ function Login () {
         <input className='pl-2 pr-2 h-8 rounded-md shadow-md border border-gray-300 focus:outline-none focus:ring-1 focus:ring-black focus:border-transparent ...' type='password' />
         <button className='font-semibold p-2 mt-6 rounded-xl shadow-md bg-yellow-300 hover:bg-yellow-400 focus:outline-none focus:ring-2 focus:ring-yellow-300 focus:ring-opacity-50 ...' type='submit'>Log In</button>
       </form>
-      <p className='text-sm text-center mt-3'>Don't have an account? <span className='text-blue-500 cursor-pointer' onClick={directToSignUp}><strong>Sign up</strong></span></p>
+      <p className='text-sm text-center mt-3'>Don't have an account? <span className='text-blue-500 cursor-pointer'><Link to='/signup'><strong>Sign up</strong></Link></span></p>
```

Comment:
`{ 'Sign up' }`

Reply:
@ashrafmahmood  Do this same change at all the other place also

Reply:
I have tried that but react standard extension undo it by saying 'Curly braces are unnecessary here'.

Reply:
Okay